openfl/component/straggler_handling_functions/cutoff_time_based_straggler_handling.py

```diff
@@ -4,30 +4,130 @@
 """Cutoff time based Straggler Handling function."""
 import numpy as np
 import time
+import threading
+from typing import Callable
+from logging import getLogger
 
-from openfl.component.straggler_handling_functions import StragglerHandlingFunction
+from openfl.component.straggler_handling_functions import StragglerHandlingPolicy
 
 
-class CutoffTimeBasedStragglerHandling(StragglerHandlingFunction):
+class CutoffTimeBasedStragglerHandling(StragglerHandlingPolicy):
     def __init__(
         self,
         round_start_time=None,
         straggler_cutoff_time=np.inf,
         minimum_reporting=1,
         **kwargs
     ):
+        if minimum_reporting <= 0:
+            raise ValueError(f"minimum_reporting cannot be {minimum_reporting}")
```

Comment:
I'd add a clarification that `minimum_reporting` is expected to be a positive integer

Reply:
Changed message to `"minimum_reporting must be >0"`.